modules/account-lib/src/coin/dot/transactionBuilder.ts

```diff
@@ -206,6 +208,8 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     this.transaction.chainName(this._material.chainName);
     if (this._keyPair) {
       this.transaction.sign(this._keyPair);
+    } else if (this._signature) {
+      this.transaction.addSignature(this._signature);
```

Comment:
This method is from the builder not transaction: #1990

Reply:
https://github.com/BitGo/BitGoJS/pull/1990/files#diff-b4a14cb39fe8cc16ac38f9ccb37aabb1e4b64c4dfe6596d39fd6f2e24437248eR375-R380